content/html/concepts/attributes/terms/title/title.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'title'
+Description: 'Contains and can display the specified information about the element it belongs to.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Attributes'
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`title`** global attribute contains and can display the specified information about the element it belongs to.
+
+> **Note:** Although the `title` attribute can display additional information on the screen, it does not work with touch screens, keyboard-only navigation, or assistive technologies.
```

Comment:
The note about assistive technologies is contradicted by the bullet point about `<iframe>` below

Reply:
Thank you, I have changed the text to resolve that!